openshift_pool/openshift/stack.py

```diff
@@ -166,11 +175,16 @@ def __init__(self, name):
         """
         self._name = name
         self._stack = None
+        Loggable.__init__(self)
 
     @cached_property
     def heat_client(self):
         return StackBuilder().heat_client
 
+    @cached_property
+    def ssh_details(self):
+        return CONFIG_DATA["ssh"]
```

Comment:
Same comment as about, lets wrap this in a try/except block.

Reply:
@jawatts  I think we have to do a major work here and create a config object that would include all this kind of configuration handling and wrap it in a better way, therefore I would prefer to keep this that way.

You can see a similar example to the one here few lines above this property in the ```openstack_details``` function

Reply:
@shalomnaim1 why are we setting this as a property? Why not just reference it when needed or set a class attribute to it on init